### ISSUE TITLE
[FEATURE] - GUI for Atlas scripts

### ISSUE THREAD

**Issue by TirOFlanc** [Atlas-OS/Atlas#1073]
### What is your feature request regarding to?

Atlas Playbook

### Is your feature request related to a problem? Please describe.

Hello,

I would like to know if there is a utility for AtlasOS like for ReviOS (https://github.com/meetrevision/revision-tool) ?

More details in this video : https://www.youtube.com/watch?v=2Ya18RAB9Z8

![image](https://github.com/Atlas-OS/Atlas/assets/56934692/339033f4-fa9f-4872-80cb-35e711d7ef43)

### Describe the solution you would like.

I'd love a great utility like Revision Tool!

### Describe alternatives you have considered.

Nop

**Comment by he3als** [comment]
Although a proper GUI is a better user experience, it's not a priority as it only changes a little functionally. However, as @PencilNavigator said, I'm sure there will be one eventually.

**Comment by JayXTQ** [comment]
It's not a "I'm sure" @he3als, it's already in the process of being made

**Comment by TirOFlanc** [comment]
Yes, it's not essential, but it will benefit the user experience.

I'm delighted that this is planned and already underway. Thank you!

I guess it'll only come out in a major update?

**Comment by he3als** [comment]
> It's not a "I'm sure" @he3als, it's already in the process of being made

I'm aware.

> I guess it'll only come out in a major update?

We don't have an ETA, but most likely.

**Comment by RadNotRed** [comment]
https://github.com/Atlas-OS/atlas-toolbox/
